src/main/java/com/google/ehub/data/CommentDataManager.java

```diff
@@ -0,0 +1,38 @@
+package com.google.ehub.data;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import java.util.*;
+
+public class CommentDataManager{
+
+  public void addItemComment(String message, long timestamp){
+    Entity commentEntity = new Entity("ItemPageComments");
```

Comment:
I suggest creating keys for all of the string values such as "ItemPageComments" and "message". 
```
private static final MESSAGE_PROPERTY_KEY = "message";
private static final TIMESTAMP_PROPERTY_KEY = "timestamp";
```